### DISCUSSION TITLE
Download files between a period of time

### DISCUSSION THREAD

**Discussion by otg688** [Dineshkarthik/telegram_media_downloader#225] [Ideas]
i want to use it to download videos from sometime to sometime

**Comment by StarWhiz** [comment]
@otg688 's idea would allow me to get a specific file so that would be much appreciated. Is there any other way to download a specific file instead of all files like by message_id? A channel I'm in disabled downloads and I'm only interested in a single file for example.

**Reply by dongle77** [reply]
Yes.  What if you don't want to download every single video, only the most recent i.e. the last week?

**Reply by amitsha1** [reply]
most old only
how to do it

**Comment by Dineshkarthik** [comment]
this is now available using the below config
- start_date - Optional: Filter messages to download only those sent after this date (ISO format). Leave `null` to disable.
- end_date - Optional: Filter messages to download only those sent before this date (ISO format). Leave `null` to disable.
